WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2262,3 +2262,21 @@ PrefRowCheckWidget          = padding:2px 0;
 PrefRowSelectWidget         = min-width:280px; width:auto; max-width:100%;
 PrefRowInlineTable          = padding:0 @PrefViewInnerSpacing@;
 CheckWidgetInputText        = padding-top:0; padding-bottom:0; height:auto; line-height:1;
+
+######################################
+#   WhiteBlackList SPECIFIC STUFF
+######################################
+
+WhiteBlackViewInnerSpacing                 = @SkinViewInnerSpacing@
+TrustedAddrPrefHeader                      = margin-bottom:@SkinWrapperPadding@
+TrustedAddrHeader                          = margin-bottom:@SkinWrapperPadding@;
+WhiteBlackViewInput                        = width:413px; border-width:1px; @MediumRoundCorners@; padding-left:@WhiteBlackViewInnerSpacing@
```

Comment:
Thanks for bringing this.
I went with earlier implementation where we were setting width on input .
Now, set this on parent container.
https://github.com/Zimbra/zm-web-client/commit/77231d0c726b0dcef692dae26f31fe56e5942136